pkg/utils/parse_arf_result.go

```diff
@@ -453,10 +484,95 @@ func remediationsFromString(scheme *runtime.Scheme, name string, namespace strin
 	return rems
 }
 
+func parseValues(remContent string, resultValues map[string]string) (string, []string, []string, error) {
+
+	t, err := template.New("").Option("missingkey=zero").Parse(remContent)
+	var valuesUsedList []string
+	var valuesMissingList []string
+	var valuesParsedList []string
+
+	if err != nil {
+		return remContent, valuesUsedList, valuesMissingList, err //Error creating template
+	}
+	buf := &bytes.Buffer{}
+	err = t.Execute(buf, resultValues)
+	fixedText := buf.String()
+
+	if err != nil {
+		// Wrongly formated remediation context
```

Comment:
Wondering if this comment is something that might be useful to include in the returned err.
Also, on this function can you remove the extra spacing?